test/http-signatures-v11/util.js

```diff
@@ -1,11 +1,27 @@
 'use strict';
 const path = require('path');
 const util = require('util');
+const fs = require('fs');
```

Comment:
Could use the `require('fs').promises` api and await the async versions below.

Reply:
what is the advantage of this over the readFileSync method?

Reply:
The advantage is that the promises API is asynchronous, while the readFileSync method is not. Promises allow you to write code in a way that is easy to read, but is still asynchronous. We should be using Promises APIs as much as we can, as they result in easier to read code w/o paying the synchronous file access cost (which is extremely high). In this particular case, it's not a big deal, but you don't want to get in the habit of using readFileSync.

Reply:
I understand promises, we actually are using async await which is a generator that yields a promise anyways did change it. Was using it out of connivence and forgot that the sync code would be blocking if the files happen to take awhile to access.